content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -251,6 +257,10 @@ sudo systemctl restart containerd
 When using kubeadm, manually configure the
 [cgroup driver for kubelet](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/#configuring-the-kubelet-cgroup-driver).
 
+In Kubernetes {{< skew currentVersion >}}, you can enable automatic detection of the
```

Comment:
Same as above.

In addition to the above confusion, using this `skew` shortcode makes it difficult
to remove contents that are no longer relevant in the future.